### PR TITLE
added 'no-warmup' arg to stress calls

### PR DESCRIPTION
**Pull request by c-kodman**
Is this as expected?

### CODE REVIEW

File: cqlsh_tests/cqlsh_copy_tests.py [hunk 1]
```diff
@@ -858,7 +858,7 @@ def test_reading_with_skip_and_max_rows(self):
         num_file_rows = 10000
 
         debug('Running stress to generate a large CSV via COPY TO')
-        self.node1.stress(['write', 'n={}'.format(num_file_rows), '-rate', 'threads=50'])
+        self.node1.stress(['write', 'n={}'.format(num_file_rows), 'no-warmup', 'no-warmup', '-rate', 'threads=50'])
```
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_copy_tests.py]: You wrote no-warmup twice

File: compaction_test.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def bloomfilter_size_test(self):
         [node1] = cluster.nodelist()
 
         for x in xrange(0, 5):
-            node1.stress(['write', 'n=100K', "no-warmup", "cl=ONE", "-rate",
+            node1.stress(['write', 'n=100K', "no-warmup", "cl=ONE", "no-warmup", "-rate",
                           "threads=300", "-schema", "replication(factor=1)",
```
Comment (knifewine) [inline compaction_test.py]: twice here too

**Comment by ptnapoleon** [comment]
Please run these on cassci and link me the results

**Comment by knifewine** [comment]
once the dupes are gone (one I saw, one Philip saw), and the tests pass, consider that a +1 from me.

File: compaction_test.py (hunk 2 above)
Comment (ptnapoleon) [inline compaction_test.py]: Nice catch

**Comment by ptnapoleon** [comment]
+1

**Pull request merged**